core/templates/css/oppia.css

```diff
@@ -2578,6 +2578,10 @@ oppia-parameter {
   width: 70%;
 }
 
+.oppia-help-modal .modal-dialog {
+  height: 330px;
+}
+
```

Comment:
Why are all these new styles added in `oppia.css`? Perhaps you could try and fit them in the modal help component html file?

Reply:
As per above discussion, set .oppia-help-modal as  `windowClass`.

But still face differ in styling of modals, you can see in proff.
Do i need to fit them in .html file by removing some classes, and make changes accordingly??

Reply:
Fix, Done